.github/workflows/docker.yml

```diff
@@ -113,13 +107,44 @@ jobs:
           docker-images: true
           swap-storage: true
 
-      - name: Build and push Docker image (tagged + versioned)
-        if: ${{ github.event_name == 'push' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+      - name: Build and push full Docker image (tagged + versioned)
+        if: ${{ (github.event_name == 'push' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') && matrix.config.full == true }}
         uses: docker/build-push-action@v6
         with:
           context: .
           push: true
           platforms: ${{ matrix.config.platforms }}
           # tag list is generated from step above
-          tags: ${{ steps.tag.outputs.output_tags }}
+          tags: full-${{ steps.tag.outputs.output_tags }}
```

Comment:
Will the old `full` tag become `full-cpu`? If yes, this will be a breaking change for downstream projects, I don't think we should let that happen

Reply:
issue is that "full" only tag came without any specification, and we have many types of builds for given targets, the possible workaround will be a hack but I think it's doable.
We have the cuda, rocm, intel, vulkan but cpu has no spec, it was just full, when you do multi-stage with the current tag system it is a bit tricky, but I will give it a try.

Reply:
we should keep the tag as-is, because many people depends on that and we currently don't have a way to communicate with them about breaking changes in tag name.

changing `full` to something else will make someone already using it never receive updates in the future. and worst, because there is no error or warning when they run it, they wouldn't know there is a breaking change.

Reply:
@ngxson the tags are as-is, just a little issue of caching layers, it's going to be a few more pushes, sorry.